cmd/kube-apiserver/app/server_test.go

```diff
@@ -32,6 +32,8 @@ func TestGetServiceIPAndRanges(t *testing.T) {
 		{"192.0.2.1/24", "192.0.2.1", "192.0.2.0/24", "<nil>", false},
 		{"192.0.2.1/24,192.168.128.0/17", "192.0.2.1", "192.0.2.0/24", "192.168.128.0/17", false},
 		{"192.0.2.1/30,192.168.128.0/17", "<nil>", "<nil>", "<nil>", true},
+		{"192.0.2.1/33,192.168.128.0/17", "<nil>", "<nil>", "<nil>", true},
```

Comment:
this is meant to be used for dual stack IIIRC,
Can you please  add test cases for IPv4, IPv6 and IPv6, IPv4 addresses?
just replace one of the subnets by 2001:db2:1:3:4::/112 per example

Reply:
add also invalid ip addresses to test it fails, ie,
"bad.ip.range,192.168.0.2/24"
"192.168.0.1" notice is missing the mask , i.e `/24`

Reply:
Ok, I fixed to include these.
- Dual stack IPv4/IPv6
- missing IPv4, IPv6 mask
- invalid IP address format

And this.
- Invalid IPv6 mask